src/Customers/CustomersClient.php

```diff
@@ -148,10 +149,11 @@ public function getInvitation(string $code, array $parameters = []): Invitation
      * @throws NotFoundException
      * @throws PublicApiClientException
      */
-    public function createInvitation(int $contactId, array $parameters = []): Invitation
+    public function createInvitation(int $contactId, string $policy, array $parameters = []): Invitation
```

Comment:
As long as the package is public, a client can compose an update at any time, so he will end up with code that no longer works because this method is not compatible with the previous one.
In this case 2 possibilities: 

- Warn the users of this package by indicating what has changed in the UPGRADING.md file and increase the minor version.

- Or create a new method (ex: createInvitationV2) and leave the previous one with depricated which will be deleted at the next upgrade

Reply:
I create a new one named : createInvitationWithPolicy